docs/content/authentication.md

```diff
@@ -5,47 +5,90 @@ title: "Authentication"
 
 # Authentication and Configuration
 
-Each system handles authentication differently. Authentication must be configured
-if resources are provided for a system.
+## Introducing Contexts
+Grizzly supports 'contexts' (much like `kubectl` contexts), allowing users to maintain
+configuration for multiple Grafana (and Mimir and Synthetic Monitoring) systems.
+
+By default, the `default` context is used. We will discuss how to use the default
+context first, then explain how to use multiple contexts thereafter.
+
+## Settings
+
+> **NOTE**: If you have used Grizzly previously with environment variables, you can
+  initialise the `default` context from your environment simply with `grr config init`.
```

Comment:
How about ` grr config init --from-env-variables`, assuming that this command would fail if the variables are not set?

Reply:
`init` might sound more like something 'fresh' while falling back to env variables might be confusing.